.stylelintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "stylelint-config-sass-guidelines",
```

Comment:
This configuration is based on [this guideline](https://sass-guidelin.es/). It is mostly [stylistic conventions](https://sass-guidelin.es/#too-long-didnt-read) that we already adopted most - if not all - part of it.